src/Slot-Core/LiteralVariable.class.st

```diff
@@ -74,7 +74,7 @@ LiteralVariable >> definitionString [
 { #category : #'code generation' }
 LiteralVariable >> emitStore: aMethodBuilder [
 	| tempName |
-	tempName := Object new.
+	tempName := '0TempForStackManipulation'.
```

Comment:
I have a question.... in this case the names should not be unique? 
Because doing Object new each time produced different objects. 
Having an string as name, it does not produce clashes?

Reply:
I don't think so:

- there is no nesting possible and thus the whole usage is just local.
- all use starts with writing into the var. Thus no confusion of old values.

So what will happen if there are multiple writes of this kind of var in the same method?
all of them will use the same synthesised var, but the use never overlaps.

At leas that is how I understand it...